tests/hwtimer_wait/main.c

```diff
@@ -30,7 +30,7 @@ int main(void)
     puts("When the race condition is hit, the timer will wait for a very very long time.");
 
     long iterations = 10000;
-    long start_duration = (HWTIMER_SPIN_BARRIER + 1) << 5;
+    long start_duration = (20 + 1) << 5;
```

Comment:
IIRC the idea was to start the test at the lower threshold of `HWTIMER_SPIN_BARRIER` so the spin function is not involved. What do you think about leaving this line as it was but decreasing the number of iterations so the test won't take so long? Or did I intercept something you discussed in the latest timer-related PRs?